src/main/java/models/ClassificationLevel.java

```diff
@@ -1,5 +1,8 @@
 package models;
 
 public enum ClassificationLevel {
-    UNCLASSIFIED, CONFIDENTIAL, SECRET, TOP_SECRET
+    UNCLASSIFIED,
```

Comment:
Please remove unused enum values (usually it depends whether we will be use them in the future, but for current case - let's assume all existing functionality is a final form and we will not need unused values). Please apply this change to ExperimentalType enum as well

Reply:
deone